### PR TITLE
op-chain-ops: Preserve predeploy nonces

### PR DESCRIPTION
**Pull request by mslipper**
Users are unable to use the OptimismMintableERC20Factory contract due to the nonce being reset to zero. This PR updates the predeploy storage wipe function to preserve nonces during the migration. This is fixable on Goerli by incrementing the nonce manually on the factory contract.

### CODE REVIEW

**Review by protolambda — COMMENTED** [review]
Change looks good, but we should probably be testing this as part of the migration checks

File: op-chain-ops/genesis/setters.go [hunk 1]
```diff
@@ -92,7 +92,13 @@ func WipePredeployStorage(db vm.StateDB) error {
 		}
 
 		log.Info("wiping storage", "name", name, "address", *addr)
+
+		// We need to make sure that we preserve nonces.
+		oldNonce := db.GetNonce(*addr)
 		db.CreateAccount(*addr)
```
Comment (tynes) [inline op-chain-ops/genesis/setters.go:98]: We should also preserve balance? I don't think any of them should have balance but just in case

Comment (mslipper) [reply op-chain-ops/genesis/setters.go:98]: Yeah good call.

**Review by protolambda — APPROVED** [review]
LGTM

**Pull request merged**